### PR TITLE
Attempt to turn off 10s reachability sleep

### PR DESCRIPTION
**Pull request by howardjohn**
[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

Pull Request Attributes

Please check any characteristics that apply to this pull request. 

[ ] Does not have any changes that may affect Istio users.

### CODE REVIEW

**Review by ericvn — APPROVED** [review]

**Comment by howardjohn** [comment]
@istio/wg-security-maintainers I ran this ~8 times with no failures, I think its hopefully safe to merge now

**Comment by howardjohn** [comment]
ping @istio/wg-security-maintainers

File: tests/integration/security/util/reachability/context.go [hunk 1]
```diff
@@ -172,11 +179,17 @@ func Run(testCases []TestCase, ctx framework.TestContext, apps *util.EchoDeploym
 								if c.Include(src, opts) {
 									expectSuccess := c.ExpectSuccess(src, opts)
 
-									subTestName := fmt.Sprintf("with scheme %s to %s:%s%s",
+									// TODO failure from naked -> VM is 1s due to the fact service is empty
```
Comment (yangminzhu) [inline tests/integration/security/util/reachability/context.go:182]: is this TODO still relevant? I saw you just skipped the naked -> VM above.

Comment (howardjohn) [reply tests/integration/security/util/reachability/context.go:182]: Probably not since we are skipping it now

Comment (yangminzhu) [reply tests/integration/security/util/reachability/context.go:182]: can you remove it? approved with /hold, feel free to merge after removing.

**Review by yangminzhu — APPROVED** [review]
/hold

**Pull request merged**